src/endian_reader.rs

```diff
@@ -62,7 +62,7 @@ pub type EndianArcSlice<Endian> = EndianReader<Endian, Arc<[u8]>>;
 /// use std::sync::Arc;
 ///
 /// /// A type that represents an `mmap`ed file.
-/// #[derive(Debug)]
+/// #[derive(Debug, Clone)]
```

Comment:
This should *not* be `Clone` since it represents the mapped region directly and calls `unmap` on drop.

Reply:
Good catch, no idea why I changed that.